frontend/public/tokener.html

```diff
@@ -19,7 +19,12 @@
       if (!error) {
         var next = localStorage.getItem('next') || '';
         if (next) {
-          localStorage.removeItem('next');
+	  if (next.substr(0,1) == "/" && json.loginSuccessURL.substr(-1) == "/") {
```

Comment:
Use `===` for comparison in JavaScript. (See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness for why)

Also, you can use `next[0]` instead of substr(0, 1). eg:`next[0] === '/'`.